vent/main.py

```diff
@@ -1,24 +1,22 @@
 #!/usr/bin/env python
+import argparse
+from vent.coordinator.ui_control_module import get_ui_control_module
 
-# This is the main script. The UI people can create this for us.
-# It calls to main_control which reads in sensor data and controls the system.
-import time
-from vent import main_control, sensors, alarms
 
-SAMPLETIME = 0.01
+def parse_cmd_args():
+    parser = argparse.ArgumentParser()
+    # TODO: maybe we should add a mode without UI display, so this would only have command line interface?
+    parser.add_argument('--simulation', help='running as simulation using virtual sensors and actuators',
+                        action='store_true')
+    parser.add_argument('--multi_process', help='running UI and controller as separate processes', action='store_true')
```

Comment:
We want to default to multi-process case, so I'd recommend the arg be --single_process to override the default

Reply:
fixed